crates/devolutions-pedm/openapi/pedm-api.yaml

```diff
@@ -342,7 +342,7 @@ components:
       properties:
         Seconds:
           type: integer
-          format: int64
+          format: uint64

```

Comment:
question: Why all the integer types are changed?

Reply:
I don't know either, I just re-generated packages as described in `README.md` 🤔